### DISCUSSION TITLE
Are AWS-Tools for Powershell being developed?

### DISCUSSION THREAD

**Discussion by mcdonamw** [aws/aws-tools-for-powershell#342] [General]
I strictly use powershell and I've just been thrown into performing some AWS administration, but I'm having a hard time getting familiar with these tools.  The documentation seems light and this github seems inactive.  Is there anyone else developing AWS based tools in Powershell, if not Amazon directly?

**Comment by afroz429** [comment]
Hello, 

AWS Tools for PowerShell is actively being developed and new versions are being released almost every day. Please see the [change logs](https://github.com/aws/aws-tools-for-powershell/blob/master/changelogs/CHANGELOG.2024.md) for the changes.

Please feel free to create any issues/discussions if needed. We are actively monitoring the issues.

**Reply by mcdonamw** [reply]
I somehow missed this reply.  At any rate, I was specifically hoping for some improvements to the Route53 cmdlets but in looking through the change logs there's not much of anything R53 related.  

Doing things like simply adding an additional record to a recordset is a nightmare.  Granted that's mostly due to the API itself, but I hoped the Powershell cmdlets would make things more simple and wrap a lot of functionality into simple methods that would obfuscate a lot of the complext API like the old cli53 tool someone built years back does, though even that's limited as well.

Ultimately r53 automation is a PITA, especially for someone new to the platform.  I don't understand why AWS would make this so difficult.

**Reply by boblodgett** [reply]
The cmdlets are wrapping the AWS SDK for .NET API calls which in turn are modeled against the service API itself. Obfuscating the calls into a high-level library or specific cmdlet would depend on customer demand and feasibility for such a feature. Opening a feature request on this repo with some usage examples would be a good way to make the request visible to us and other customers as well.
